### PR TITLE
Add management command for costs report pre-validation

### PR DESCRIPTION
**Pull request by mkurek**

### CODE REVIEW

File: src/ralph_scrooge/management/commands/_utils.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import argparse
+from datetime import datetime
+
+
+def valid_date(date_):
```
Comment (xor-xor) [inline src/ralph_scrooge/management/commands/_utils.py]: It is in utils already: https://github.com/allegro/ralph_pricing/blob/develop/src/ralph_scrooge/utils/common.py#L130 ;) What about importing it as `valid_date` to your mgmt command..?

**Review by xor-xor — CHANGES_REQUESTED** [review]

File: src/ralph_scrooge/management/commands/_utils.py (hunk 1 above)
Comment (mkurek) [reply src/ralph_scrooge/management/commands/_utils.py]: True, I'll fix it :)

Comment (mkurek) [reply src/ralph_scrooge/management/commands/_utils.py]: done

File: src/ralph_scrooge/management/commands/costs_report_prevalidation.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from django.core.management.base import BaseCommand
+
+from ralph_scrooge.utils.common import validate_date
```
Comment (xor-xor) [inline src/ralph_scrooge/management/commands/costs_report_prevalidation.py]: s/validate_date/validate_date as valid_date/

File: src/ralph_scrooge/management/commands/costs_report_prevalidation.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+
+class Command(BaseCommand):
+    """
+    Validate data for costs report for particular date.
+    """
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--date',
+            type=validate_date,
```
Comment (xor-xor) [inline src/ralph_scrooge/management/commands/costs_report_prevalidation.py]: s/validate_date/valid_date/

File: src/ralph_scrooge/management/commands/detect_cycles.py [hunk 4]
```diff
@@ -5,31 +5,20 @@
...
-
 class Command(BaseCommand):
     """
     Detect if there is cycle in charging between PricingServices for given date
     """
     def add_arguments(self, parser):
         parser.add_argument(
             '--date',
-            type=valid_date,
+            type=validate_date,
```
Comment (xor-xor) [inline src/ralph_scrooge/management/commands/detect_cycles.py]: s/validate_date/valid_date/

File: src/ralph_scrooge/management/commands/detect_cycles.py [hunk 5]
```diff
@@ -5,31 +5,20 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import argparse
-from datetime import datetime
-
 from django.core.management.base import BaseCommand
 
+from ralph_scrooge.utils.common import validate_date
```
Comment (xor-xor) [inline src/ralph_scrooge/management/commands/detect_cycles.py]: s/validate_date/validate_date as valid_date/

**Review by xor-xor — CHANGES_REQUESTED** [review]

**Comment by mkurek** [comment]
@xor-xor fixed all comments

**Review by xor-xor — APPROVED** [review]

**Pull request merged**